index.js

```diff
@@ -25,13 +26,17 @@ function handleMessages(file, messages) {
 	}
 
 	messages.forEach(function (message) {
+		if (message.type === 'info' && !options.showInfo) {
+			return;
+		}
+
 		if (message.type === 'error') {
 			success = false;
 		}
 
-		var type = (message.type === 'error') ? errorText : warningText;
+		var type = (message.type === 'error') ? errorText : ((message.type === 'info') ? infoText : warningText);
 
-		var location = 'Line ' + message.lastLine + ', Column ' + message.lastColumn + ':';
+		var location = 'Line ' + (message.lastLine || 0) + ', Column ' + (message.lastColumn || 0) + ':';
```

Comment:
They are hidden, if both of them are undefined (see line -58 +63..67), 0 is only used when one of them is undefined. Ok, will change it to "unknown" for that case.